tests/src/rules/default.js

```diff
@@ -28,19 +28,19 @@ ruleTester.run('default', rule, {
 
     // es7 export syntax
     test({ code: 'export bar from "./bar"'
-         , parser: 'babel-eslint' }),
+         , parser: require.resolve('babel-eslint') }),
```

Comment:
why is this needed? eslint should be resolving whatever’s passed to parser

Reply:
https://eslint.org/docs/user-guide/migrating-to-6.0.0#-ruletester-now-requires-an-absolute-path-on-parser-option

If I remove it I get an error in line with the above:
```
1) default valid export bar from "./bar":

      AssertionError [ERR_ASSERTION] [ERR_ASSERTION]: Parsers provided as strings to RuleTester must be absolute paths
      + expected - actual

      -false
      +true
```